src/content/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-8-13-2.mdx

```diff
@@ -0,0 +1,15 @@
+---
+subject: Node.js agent
+releaseDate: '2022-05-31'
+version: 8.13.2
+downloadLink: 'https://www.npmjs.com/package/newrelic'
+---
+
+## Notes
+
+* Upgraded `protobufjs` to resolve [CVE-2022-25878](https://cve.mitre.org/cgi-bin/cvename.cgi?name=2022-25878)
+
+
+### Support statement:
+
+* New Relic recommends that you upgrade the agent regularly to ensure that you're getting the latest features and performance benefits. Additionally, older releases will no longer be supported when they reach [end-of-life](/docs/using-new-relic/cross-product-functions/install-configure/notification-changes-new-relic-saas-features-distributed-software).
```

Comment:
you also missed updating this one if we want the new URL. personally I don't think this really matters as it redirects accordingly

Reply:
Yes, you're right. It doesn't matters